docs/ed-fi-badges/3-registry-of-ed-fi-badged-products-and-services.md

```diff
@@ -0,0 +1,69 @@
+---
+author: Eric Jansson
+tags: [education Agency,v]
+description: Requirements for technology providers to be eligible for Ed-Fi badges.
+created: 2024-08-07
+title: Registry of Ed-Fi Badged Products and Services
+---
+# Registry of Ed-Fi Badged Products and Services
+:::tip
+
+🚧 This site is under construction. In the meantime, please see
+[Ed-Fi API Guidelines](https://edfi.atlassian.net/wiki/spaces/EFAPIGUIDE/overview)
+for more information.
+
+:::
+
+This page lists products that have been awarded an Ed-Fi badge.
+
+## Ed-Fi Managed ODS / API Platform Badge
+
+| Product Image | Product | Provider | Provider Website | Product Information | Badge Valid Through | Verifying Agencies | Version Compatibility | Product Availability |
+|---------------|---------|----------|------------------|----------------------|---------------------|--------------------|-----------------------|----------------------|
+| !EdGraphDarkLogo | EdGraph | EdWire | edgraph.com | edgraph.com | August 28, 2025 | NEFEC, Sumter County Schools | Ed-Fi ODS API for Suite 3 v3.4, Ed-Fi ODS API for Suite 2 v2.5 and v2.6 | Available to any agency (all sizes) or vendor in the U.S. |
+| !EA-startingblocks-color | StartingBlocks | Education Analytics | edanalytics.org | edanalytics.org/products/starting-blocks | May 12, 2025 | South Carolina DOE, Texas Education Exchange, Vermont Agency of Education | EdFi ODS/API versions 6.x and 7.x, Admin API 1.x and 2.x, Data Standards 4 and 5 | Available to any agency (all sizes) in the U.S. |
+| !k12analytics.png) | K12Analytics | K12Analytics Engineering | k12analyticsengineering.dev | k12analyticsengineering.dev | March 8, 2025 | BreakThrough Schools | Ed-Fi ODS API for Suite 3 v3.1.1 | Available to any agency (all sizes) in the U.S. |
+| !ESPG Stacked-Logo | ESP Solutions Group | ESP Solutions Group | espsolutionsgroup.com | espmsp.com | May 18, 2025 | Wyoming Department of Education (WDE) | Ed-Fi ODS API 2.x and 3.x | Available to any agency (all sizes) in the U.S. |
+| !Edufied_Logo.jpeg) | Edufied Hosting | edufied | edufied.com | edufied.com/about-us | August 30, 2025 | Holdsworth Center, TX, Boston Public Schools, MA, New Mexico PED, NM | Ed-Fi ODS / API Suite 3 v5.3, Ed-Fi ODS / API Suite 3 v7.0, Commitment to continuous support for most recent versions | Available to any agency (all sizes) in the U.S. |
+| !ClassLink Horizontal Logo | ClassLink OneData | ClassLink | classlink.com | classlink.com/products | September 12, 2025 | Clay County, Ennis ISD | Ed-Fi ODS / API for Suite 3 v6.1 | Available to any agency (all sizes) in the U.S. |
```

Comment:
With a big table like this, we'll want to use MDX so that we can more easily manage the content in JSON. Example: [certification registry](https://github.com/Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io/blob/main/partners/2-certification/registry-of-ed-fi-certified-products.mdx).